tsdb/points.go

```diff
@@ -739,14 +739,19 @@ func skipWhitespace(buf []byte, i int) int {
 func scanLine(buf []byte, i int) (int, []byte) {
 	start := i
 	quoted := false
+	fields := false
```

Comment:
Mild preference for this to be called something like "processingFields", but I'll leave that you, as you know what is most consistent with the rest of this source.